src/database/index.tsx

```diff
@@ -329,25 +329,74 @@ export const getFacilityModelVersions = async ({
   }
 };
 
+const getUserDocument = async (params: {
+  email?: string;
+  auth0Id?: string;
+}): Promise<firebase.firestore.DocumentData | never> => {
```

Comment:
slightly pedantic (and possibly incorrect) but I think if you type this as `{email: string} | {auth0Id: string}` you will enforce only one of these being set at a time; could avoid confusion since we are just arbitrarily picking whichever key JavaScript decides should be first. You might not even need to throw an error in that case because the static analysis would cover it? (fine with me if you just ignore this though since I'm not 100% sure it will work anyway)

Reply:
Seems to have worked 👍